### ISSUE TITLE
Error on import 'effector-storage/local' 

### ISSUE THREAD

**Issue by syabro** [yumauri/effector-storage#12]
```
ERROR in ./node_modules/effector-storage/local/index.js 1:58-95
Module not found: Error: Can't resolve '../storage' in '/xxx/node_modules/effector-storage/local'
Did you mean 'index.js'?
BREAKING CHANGE: The request '../storage' failed to resolve only because it was resolved as fully specified
(probably because the origin is a '*.mjs' file or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
  xxx
 @ ./src/app.tsx 3:0-41 15:15-20
 @ ./src/index.tsx 2:0-26 4:9-12
```

**Comment by syabro** [comment]
Solved by adding to webpack config
```
           {
                test: /\.m?js/,
                resolve: {
                    fullySpecified: false
                }
            },
```

https://github.com/graphql/graphql-js/issues/2721#issuecomment-723008284

(In case if someone would stuck on the same problem)

**Comment by yumauri** [comment]
Thank you for reporting this!
From the attached link I assume you are using Webpack 5, correct?
This is strange, actually, because `/storage/` has [package.json](https://npmfs.com/package/effector-storage/4.0.1/storage/package.json) file inside, with entry point for `"module"`, and Webpack should respect it...
I'll try to investigate this.

**Comment by syabro** [comment]
@yumauri yup, webpack 5.

**Comment by yumauri** [comment]
I've published version 4.0.2, should fix the issue.
Didn't try on different bundlers though, but Webpack 5 looks like working fine without any config adjustments.
@syabro Can you try new version, please?

**Comment by syabro** [comment]
@yumauri seems fine. Thanks.

**Comment by yumauri** [comment]
To be sure this case is closed, I tried to make an extended investigation with different popular bundlers.
Now I can be at least pretty sure all major bundlers are working fine with my module :)

Bundler | 4.0.2 | 4.0.3
-------- | ------|-------
Webpack 1 (babel 6 loader + es2015 preset + effector/compat) | ✅ | ✅
Webpack 2 (babel 7 loader + env preset) | ✅ | ✅
Webpack 3 (babel 7 loader + env preset) | ✅ | ✅
Webpack 4 | ✅ | ✅
Webpack 5 | ✅ | ✅
Parcel 1 | ✅ | ✅
Parcel 2 | ✅ | ✅
Rollup | ✅ | ✅
Snowpack | ✅ | ✅
esbuild | ✅ | ✅
FuseBox 3 | ❌ | ✅
FuseBox 4 | ✅ | ✅
Browserify (with `require` syntax) | - | ✅
Poi | - | ✅

There was an issue with FuseBox 3, it has hardcoded `.js` extension to lookup files, so `.cjs` files weren't working, and esm weren't working also, I don't know why...
So, I've published version 4.0.3, the only difference is that CommonJS modules now have an extension `.cjs.js`

**Comment by yumauri** [comment]
In the version 4.2.0 I've decided to change `.cjs.js` extensions back to `.cjs`. I've tested all bundlers, and everything is working fine, aside from FuseBox3. [Which I abandon](https://github.com/yumauri/effector-storage/issues/16#issuecomment-811110447).
